pytest_mh/_private/plugin.py

```diff
@@ -63,6 +65,7 @@ def __init__(self, pytest_config: pytest.Config) -> None:
         self.mh_collect_artifacts: MultihostArtifactsMode = pytest_config.getoption("mh_collect_artifacts")
         self.mh_artifacts_dir: Path = Path(pytest_config.getoption("mh_artifacts_dir"))
         self.mh_compress_artifacts: bool = pytest_config.getoption("mh_compress_artifacts")
+        self.mh_ignore_preferred_topology: bool = pytest_config.getoption("mh_ignore_preferred_topology")
```

Comment:
print value in https://github.com/next-actions/pytest-mh/blob/master/pytest_mh/_private/plugin.py#L202